<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2080</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>2041</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="save_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить изменения</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить элементы</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="id_box">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите id кофе</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>430</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить кофе</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
